concepts/index.md

```diff
@@ -6,29 +6,38 @@ layout: documentation
+**Things** are the entities that can be physically added to a system.
+These Things may provide more than just one function (for example, a thermostat may allow you to set the room temperature and also turn on a fan).
+It is important to note that Things do not have to be physical devices; they can also represent a web service or any other manageable source of information and functionality.
 
-**Items** represent (fine-grained) functionality that is used by applications - as user interfaces or automation logic. Items have a state and they can receive commands.
+Things provide their functionality through a set of **Channels**.
+Channels are "passive" and can be regarded as a declaration of a Thing; what it can offer.
+Whether an openHAB installation takes advantage of a particular capability offered through a Thing's Channel depends on whether it has been configured to do so.
+openHAB makes use of Channels through links you establish to Items when you configure your system (see below).
 
-The glue between Things and Items are **Links**. Links are associations between exactly one Thing Channel and one Item.
-If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is handled through the given Channel.
+**Items** represent (fine-grained) functionality that can be used by applications, either in user interfaces or automation logic.
+Items have a state and they can receive commands.
+
+The glue between Things and Items are **Links**.
+Links are associations between exactly one Thing Channel and one Item.
+If a Channel is linked to an Item, it is "enabled", which means that the functionality that the Item represents is accessible through the Channel.
 Channels can be linked to multiple Items and Items can be linked to multiple Channels.
 
-To illustrate these concepts, take a two-channel actuator that controls two lights:
+To illustrate these concepts, consider the example below of a two-channel actuator that controls two lights:
 
 ![](images/thing-devices-1.png)
 
-The actuator is the _Thing_. This might be installed in the electrical cabinet, it has a physical address and needs to be setup and configured in order to be used.
-The user is instead interested in the two lights, which are located at different locations in his home. These lights are the desired functionality, thus the _Items_ and they are linked to the _Channels_ of the actuator.
-A _Link_ can be regarded like a physical wire in this example.
-
-
-
-
+The actuator is a *Thing* that might be installed in an electrical cabinet.
+It has a physical address and it must be configured in openHAB in order to be used (remember the physical view introduced at the beginning of this article?).
 
+Functionally, the user is interested in controlling the two lights.
+In order for the user to achieve the desired functionality with the lights, the *Items* which represent the lights in openHAB are linked to the *Channels* of the actuator.
```

Comment:
Consider changing this sentence to a single item explanation.

Something like:
"In order for the user to achieve the desired functionality with the lights, each light is linked as a Item to one of the actuators *Channels*."

Reply:
How about the following, which I am putting in my next commit?

The user wants to control the two lights.
In order to achieve this functionality, two *Items* representing the lights are linked to *Channels* of the actuator, the *Thing*.
In this case, one can think of a *Link* being equivalent to a physical wire.